### PR TITLE
Use the internal hadoop

### PR DESCRIPTION
**Pull request by merlintang**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Comment by merlintang** [comment]
@dongjoon-hyun @bikassaha  please review this.

**Comment by dongjoon-hyun** [comment]
For this purpose, please use this branch.

https://github.com/hortonworks-spark/spark-llap/tree/HDP-2.6.0.10

It's because 2.6.0.10-21 is not a final. Previsouly, I made HDP-2.6.0.x branch and used that.

File: build.sbt [hunk 1]
```diff
@@ -146,7 +146,7 @@ addArtifact(artifact in (Compile, assembly), assembly)
 
 resolvers += "Local Maven Repository" at "file://"+Path.userHome.absolutePath+"/.m2/repository"
 resolvers += "Additional Maven Repository" at repoUrl
-resolvers += "Hortonworks Maven Repository" at "http://repo.hortonworks.com/content/groups/public/"
+resolvers += "Hortonworks Private Maven Repository" at "http://nexus-private.hortonworks.com:8081/nexus/content/repositories/IN-QA/"
```
Comment (dongjoon-hyun) [inline build.sbt]: Don't change the line 149. Please add a new line for private one.

**Comment by dongjoon-hyun** [comment]
+1 except two minor comments. Please commit directly on that branch.
FYI, if you want, you can publish the artifact from your laptop to internal repo directly, too.

**Pull request closed without merging**